nrf_rpc/include/nrf_rpc_cbor.h

```diff
@@ -124,6 +124,14 @@ struct nrf_rpc_cbor_ctx {
 		_nrf_rpc_cbor_prepare((struct nrf_rpc_cbor_ctx *)(&(_ctx)), (_len) + 1); \
 	} while (0)
 
+/** @brief Checks memory for a packet.
+ * 
+ * Mecro checks whether memory for a packet has been allocated.
+ * 
+ * @param[in] _ctx Context allocated by @ref NRF_RPC_CBOR_ALLOC.
+ */
+#define NRF_RPC_CBOR_IS_ALLOC(_ctx) ((_ctx).out_packet != NULL)
```

Comment:
What is the purpose of this change? It seems unrelated to the main change. If it's a standalone API improvement, you should move it to a separate commit.

Reply:
Buffor allocation is done by RPC transport function. If the transport is not properly initialized the buffer allocation macro - `NRF_RPC_CBOR_ALLOC`, does not return error. Therefore, it is a helper macro. 

In addition, there is also macro `NRF_RPC_GROUP_STATUS` to check whether the group/transport was initialized properly. If not then the user should not use such a group.